packrat/src/test/java/com/github/reflectoring/infiniboard/packrat/OverseerTestApplication.java

```diff
@@ -0,0 +1,11 @@
+package com.github.reflectoring.infiniboard.packrat;
+
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+/**
+ * Spring Boot application for overseer tests
+ */
+@SpringBootApplication
+public class OverseerTestApplication {
+
+}
```

Comment:
Let's always end files with OR without newline and document this style in our wiki. I prefer ending with a newline. But let us discuss this next meeting.